src/components/arbeidsforhold/Arbeidsforhold.tsx

```diff
@@ -73,7 +85,12 @@ export function Arbeidsforhold(props: IFaktum<IQuizGeneratorFaktum>) {
         );
 
         return (
-          <div key={svarIndex}>
+          <div
```

Comment:
I stedet for å repetere samme useEffect i alle faktum, hadde det vært en ide og kjøre denne logikken i Faktum.tsx og sende inn forwardRef prop ned til hvert faktum?

Reply:
Godt poeng, skal sjekke ut hvordan det gjøres.

Reply:
Fixed. Ligger noen få useEffect logikken, men fikk fjerne ganske mange der 👍